### PR TITLE
add bzip2 to common-utils (former https://github.com/microsoft/vscode-dev-containers/pull/1586)

### PR DESCRIPTION
**Pull request by lucasfcnunes**

### CODE REVIEW

File: src/common-utils/install.sh [hunk 1]
```diff
@@ -85,6 +85,7 @@ if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
         rsync \
         ca-certificates \
         unzip \
+        bzip2 \
```
Comment (samruddhikhandale) [inline src/common-utils/install.sh:88]: Can we bump the minor version here ( to 1.1.0 ) as we are adding a new package? (We follow semantic versioning) https://github.com/devcontainers/features/blob/main/src/common-utils/devcontainer-feature.json#L3

Features are released every time a PR is merged, these changes won't take effect unless the version specified here is different than the ones already published 😄

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks! 
Requested one change, besides that LGTM.

image

`bzip2` package will consume approx 196 kB of extra space.

File: src/common-utils/install.sh (hunk 1 above)
Comment (lucasfcnunes) [reply src/common-utils/install.sh:88]: 👍🏽

**Review by samruddhikhandale — APPROVED** [review]
Thanks!

**Pull request merged**